client/src/components/new-request/search.vue

```diff
@@ -286,11 +288,8 @@ export default class NewSearch extends Vue {
     'AML',
     'CHG'
   ]
-  entityBlurbs (entity_type_cd: string) {
-    if (newReqModule.entityBlurbs) {
-      return newReqModule.entityBlurbs?.find(type => type.value === entity_type_cd)?.blurbs || ''
-    }
-    return ''
```

Comment:
Seems odd that this was OK before it is an array of strings. Although, i guess you wouldn't be fixing it if it was..